Add Cancel Query section to DBM docs (#29688)
DataDog/documentation: content/en/database_monitoring/database_hosts.md

{{< img src="database_monitoring/db-list-active-connections-2.png" alt="The Active Connections tab of the details panel for a single database host on the Databases page" style="width:90%;" >}}

### Cancel a live query

To cancel a long running database query directly from Datadog:
1. In **Active Connections**, click **Cancel Query**:
   {{< img src="database_monitoring/db-cancel-query-button.png" alt="The Cancel Query button inside the Active Connections panel" style="width:90%;" >}}
1. Select a connection and click **Run Action**:
   {{< img src="database_monitoring/db-cancel-query-modal.png" alt="The Cancel Query modal when opened" style="width:90%;" >}}

The query is canceled after the run completes.

> For information on enabling Datadog to cancel queries in your database (Postgres only), see [Setup Cancel Query integration][2]. Cancelling live queries is only available for Postgres.

Reviewer: ```suggestion
```

{{< img src="database_monitoring/db-list-active-connections-2.png" alt="The Active Connections tab of the details panel for a single database host on the Databases page" style="width:90%;" >}}

### Cancel a live query

> You can cancel a long running database query directly from Datadog, here is how.
> 
> In Active Connections, click on the "Cancel Query" button:
> 
> {{< img src="database_monitoring/db-cancel-query-button.png" alt="The Cancel Query button inside the Active Connections panel" style="width:90%;" >}}
> 
> Then Select a Connection and click on Run Action:
> 
> {{< img src="database_monitoring/db-cancel-query-modal.png" alt="The Cancel Query modal when opened" style="width:90%;" >}}
> 
> Once the run completes, the query will be cancelled.
> 
> See [the setup section][2] on how to enable Datadog to cancel queries in your database (Postgres only).

Reviewer: ```suggestion
To cancel a long running database query directly from Datadog:
1. In **Active Connections**, click **Cancel Query**:
   {{< img src="database_monitoring/db-cancel-query-button.png" alt="The Cancel Query button inside the Active Connections panel" style="width:90%;" >}}
1. Select a connection and click **Run Action**:
   {{< img src="database_monitoring/db-cancel-query-modal.png" alt="The Cancel Query modal when opened" style="width:90%;" >}}

The query is canceled after the run completes.

For information on enabling Datadog to cancel queries in your database (Postgres only), see [Setup Cancel Query integration][2]. Cancelling live queries is only available for Postgres.
```

{{< img src="database_monitoring/db-list-configuration.png" alt="The Configuration tab of the details panel for a single database host on the Databases page" style="width:90%;" >}}

[1]: https://app.datadoghq.com/databases

> [2]: /database_monitoring/setup_postgres/advanced_configuration/#setup-cancel-query-integration

Reviewer: ```suggestion
[2]: /database_monitoring/setup_postgres/advanced_configuration/#set-up-cancel-query-integration
```
